unison-cli/unison/Main.hs

```diff
@@ -277,7 +298,7 @@ runTranscripts renderUsageInfo shouldFork shouldSaveTempCodebase mCodePathOption
   progName <- getProgName
   transcriptDir <- prepareTranscriptDir shouldFork mCodePathOption
   completed <-
-    runTranscripts' (Just transcriptDir) transcriptDir args
+    runTranscripts' progName (Just transcriptDir) transcriptDir args
```

Comment:
Does this print the right thing? wrt `TR.TranscriptRunFailure` or does it print "Run `transcripts --codebase foo` to do more work with it?

Reply:
It does the right thing 👍🏼 

This part is the `ucm` Main.hs file, the `transcripts` executable doesn't print `Run transcripts --codebase foo to do more work with it`, it cleans up all of its temporary codebases, if you want to debug an individual transcript you can run the failing ones manually using `ucm transcript`.

The previous version didn't include that command in the transcripts output either since it would make the transcript output change on every run 😄

Reply:
Cool thanks 👍